release-notes/release-notes-v0.5.0.md

```diff
@@ -0,0 +1,433 @@
+configure the node properly first, you can find example
+configurations [here](https://bitcoin-s.org/docs/config/configuration#example-configuration-file).
+
+You can also unzip the `bitcoin-s-cli-0.5.0.zip` folder and start using the `bitcoin-s-cli` like this:
+
+```bashrc
+./bin/bitcoin-s-cli --help
+Usage: bitcoin-s-cli [options] [<cmd>]
+
+  -n, --network <value>    Select the active network.
+  --debug                  Print debugging information
+  --rpcport <value>        The port to send our rpc request to on the server
+  -h, --help               Display this help message and exit
+
+```
+
+For more information on what commands `bitcoin-s-cli` supports check the documentation, here is where to
+start: https://bitcoin-s.org/docs/next/applications/server#server-endpoints
+
+## Verifying signatures
+
+This release is signed with [Chris's signing key](https://bitcoin-s.org/docs/next/security#disclosure) with
+fingerprint `339A49229576050819083EB3F99724872F822910`
+
+To do the verification, first hash the executable using `sha256sum`. You should check that the result is listed in
+the `SHA256SUMS.asc` file next to its file name. After doing that you can use `gpg --verify` to authenticate the
+signature.
+
+Example:
+
+UPDATE ME!!!!!!!
+```
+$ sha256sum bitcoin-s-server-0.5.0.tgz
+aa1084edb5fcd3d1dbcafe0d0fba787abf4cd455bbe38809bd9a65a49c0cd0eb bitcoin-s-server-0.5.0.tgz
+$ gpg --verify SHA256SUMS.asc 
+gpg: Signature made Thu 24 Sep 2020 12:49:39 PM CDT
+gpg:                using RSA key 339A49229576050819083EB3F99724872F822910
+gpg:                issuer "stewart.chris1234@gmail.com"
+gpg: Good signature from "Chris Stewart <stewart.chris1234@gmail.com>"
+```
+
+### Website
+
+https://bitcoin-s.org/
+
+### Releases
+
+https://repo1.maven.org/maven2/org/bitcoin-s/
+
+#### Snapshot releases
+
+https://oss.sonatype.org/content/repositories/snapshots/org/bitcoin-s/
+
+## Configuration option changes required
+
+It is required that all wallets pre #2217 need to have the configuration
+
+> bitcoin-s.wallet.aesPassword = "changeMe"
+
+See [issue #2245](https://github.com/bitcoin-s/bitcoin-s/issues/2245) for the errors that are thrown when this isn't set.
+
+This password can be changed now with the `keymanagerpassphrasechange` RPC command.
+
+## Multi Wallet Support
+
+Initial support for saving multiple seeds and wallets has been added to Bitcoin-S!
+This does not yet allow you to load and use multiple wallets at once, but you can switch between wallets using the
+`bitcoin-s.wallet.walletName` config option.
+
+## Module Level Changes
+
+### Bitcoind RPC Client
+
+The Bitcoind RPC Client now has support for v0.20!
+
+The Bitcoind RPC Client now has support for using it with a bitcoind that has multiple wallets loaded, this is done by
+setting the `walletName` parameter in the required functions.
+
+#### Commits
+
+3f18ba18d7e Bitcoind Version from String (#2421)
```

Comment:
The commit formatting seems to be broken https://github.com/bitcoin-s/bitcoin-s/blob/ac4746874f83f0e69ff250a55d097b94554afe3d/release-notes/release-notes-v0.5.0.md#commits